src/java/org/apache/cassandra/db/marshal/ValueAccessor.java

```diff
@@ -130,6 +135,8 @@ default int sizeWithShortLength(V value)
      */
     default boolean isEmpty(V value)
     {
+        if (value == null)
+            System.out.println("oops");
```

Comment:
nit: Maybe just an `assert` w/ a reasonable message?

Reply:
Oh that isn't supposed to be there at all... it's for settings breakpoint. Removed.